
path_point path_point

float32 v

float32 a

float32 yaw_rate

float32 relative_time
